docs/modules/ROOT/pages/reference/extensions/grpc.adoc

```diff
@@ -72,3 +118,34 @@ To avoid compilation issues, the `grpc-java` project recommendation is to add a
 
 Alternatively, you can use Maven or Gradle plugins to iterate over the generated code (found at `target/generated-sources` for Maven) and replace all instances of `@javax.annotation.Generated` with `@jakarta.annotation.Generated`.
 
+[id="extensions-grpc-usage-accessing-classpath-resources-in-native-mode"]
+=== Accessing classpath resources in native mode
+
+The gRPC component has various options where resources are resolved from the classpath:
+
+* `keyCertChainResource`
+* `keyResource`
+* `serviceAccountResource`
+* `trustCertCollectionResource`
+
+When using these options in native mode, you must ensure that any such resources are included in the native image.
+
+This can be accomplished by adding the configuration property `quarkus.native.resources.includes` to `application.properties`.
```

Comment:
Could we remove lines 133-140 to ease long term maintenance ? Everything is already described in embedding-`resource-in-native-executable[native mode guide]` at line 141.

Reply:
I figured it was better to have some instructions inline, because it's nicer to be able to directly copy a code snippet, rather than click away to another page and loose the context of the thing you're interested in (gRPC).

If you think not having it is better, then I can change it. Although, there are other extensions where the docs are like this, so please open an issue so we clean up the rest of the codebase.

Reply:
There are always tradeof in such questions. The justification looks fair, let's keep it.